packages/core-components/src/components/product-list/ProductItem.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import React from 'react';
+
+export type ProductItemType = {
+  productId: string;
+  productName?: string;
+  previewImage?: string;
+  description?: string;
+  specifications?: string;
+};
+
+type ProductItemProps = {
+  productItem: ProductItemType;
+  classItem?: string;
+  classImg?: string;
+  classContent?: string;
+  onClick?: (id: string) => void;
+};
+
+const ProductItem: React.FC<ProductItemProps> = ({
+  productItem,
+  classItem = 'flex flex-col',
+  classImg = 'basis-full object-contain',
+  classContent = 'basis-full',
+  onClick,
+}) => {
+  return (
+    <>
+      <div
+        id={productItem.productId}
```

Comment:
Can we add some sort of prefix for id attr here, a?

Reply:
added